backend/tests/v5/scripts/scene/removeIncompleteRevisions.test.js

```diff
@@ -38,6 +38,8 @@ const { stringToUUID, UUIDToString, generateUUID } = require(`${src}/utils/helpe
 const { count, findOne, insertMany } = require(`${src}/handler/db`);
 const Path = require('path');
 
+const { modelTypes } = require(`${src}/models/modelSettings.constants`);
```

Comment:
I think this test (and the associated script it's testing) needs modifying to remove incomplete drawings as well.